cmd/smith/main.go

```diff
@@ -58,6 +58,9 @@ func runWithContext(ctx context.Context) error {
 	if *scUrl != "" {
 		scConfig := *config // shallow copy
 		scConfig.Host = *scUrl
+		scConfig.TLSClientConfig = rest.TLSClientConfig{
+			Insecure: true,
+		}
```

Comment:
I got an error message about the cert being valid for some 10.* address (whereas I'm running smith outside kube targeting it at 192.168...).

Didn't bother investigating further, since the 'recommended' solution for service catalog is to skip-tls-verify:

https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/walkthrough.md

(search for skip-tls-verify)